packages/bson-transpilers/test/yaml/native.yaml

```diff
@@ -811,6 +848,7 @@ tests:
           java: Double.parseDouble("2")
           csharp: int.Parse("2")
           object: Number(2)
+          php: "(int) '2'"
```

Comment:
Not all languages. The most recent ones (Go language converter, Rust converter) makes Int parsing. And this is reasonable, because we have a string number without dots. So there is no reason to make float casting, as we have some integer value.

Reply:
OK to resolve.